rotkehlchen/api/v1/resources.py

```diff
@@ -1235,11 +1235,8 @@ def put(
         )
 
     @use_kwargs(delete_schema, location='json')
-    def delete(self, validator_index: Optional[int], public_key: Optional[str]) -> Response:
-        return self.rest_api.delete_eth2_validator(
-            validator_index=validator_index,
-            public_key=public_key,
-        )
+    def delete(self, validators: List[Dict]) -> Response:
```

Comment:
nitpick: Here and elsewhere can use :`Dict[str, Any]` instead of just `Dict`.

```suggestion
    def delete(self, validators: List[Dict[str, Any]]) -> Response:
```